### PR TITLE
feat: support legacy style TRAMPOLINE_BUILD_FILE

### PR DESCRIPTION
**Pull request by tmatsuo**
This PR will make it easy to migrate from Trampoline V1.

### CODE REVIEW

File: trampoline_v2.sh [hunk 1]
```diff
@@ -285,6 +285,11 @@ do
     fi
 done
 
+# We want to support legacy style TRAMPOLINE_BUILD_FILE used with V1
+# script: e.g. "github/repo-name/.kokoro/run_tests.sh"
+TRAMPOLINE_BUILD_FILE="${TRAMPOLINE_BUILD_FILE#github/[^/]*/}"
```
Comment (tbpg) [inline trampoline_v2.sh]: Might want to do `+` instead of `*`, but it probably doesn't matter.

**Review by tbpg — APPROVED** [review]

File: trampoline_v2.sh (hunk 1 above)
Comment (tmatsuo) [reply trampoline_v2.sh]: done

**Comment by tmatsuo** [comment]
@tbpg The parameter expansion uses the same rule as for file globbing. I think the newest commit has the right pattern. PTAL

File: MIGRATING_FROM_V1.md [hunk 2]
```diff
@@ -191,9 +191,13 @@ allowed_env_vars: {
 
 With Trampoline V1, `TRAMPOLINE_BUILD_FILE` is in the form of
 `github/getting-started-python/.kokoro/.*`, but with Trampoline V2,
-you can not have `github/getting-started-python` part.
+you can omit `github/getting-started-python` part.
 
-So you may need to relax the regex if you have a strict one.
+This is cleaner because it is CI agnostic, but note that Trampoline V2
+also support the legacy form too.
```
Comment (tbpg) [inline MIGRATING_FROM_V1.md]: supports

**Review by tbpg — APPROVED** [review]

File: MIGRATING_FROM_V1.md (hunk 2 above)
Comment (tmatsuo) [reply MIGRATING_FROM_V1.md]: done

**Pull request merged**